pmm/pmm2-submodules-rewind.groovy

```diff
@@ -16,15 +16,12 @@ pipeline {
             steps {
                 deleteDir()
 
-                git branch: 'PMM-2.0', credentialsId: 'GitHub SSH Key', poll: false, url: 'git@github.com:Percona-Lab/pmm-submodules'
```

Comment:
This fixes the failure of Jenkins' native git tool (reason of failure is unknown so far). Check [slack](https://perconacorp.slack.com/archives/G4VC14WTF/p1707062344085489) for more details.